index.html

```diff
@@ -502,29 +502,39 @@ <h2 class="section-heading">Contact</h2>
 		</div>
 	</footer>
 
+	<!-- Library JavaScript -->
+	<!--<script src="dist/js/lib.min.js"></script>-->
+
 	<!-- jQuery -->
+	<!-- <script src="js/jquery.js"></script>-->
+	<!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script> -->
```

Comment:
We might as well get rid of the old one since there will be no going back, right?